### PR TITLE
Added logick for EyeCore. Modified Classification to accommodate new standard for Fundus photography 

### PR DESCRIPTION
**Pull request by aldum1**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
   "name": "dicom-metadata-classification",
   "label": "Metadata Classification: DICOM",
   "description": "This gear classifies DICOM images and updates the classification metadata of the input file",
-  "version": "1.6.1",
+  "version": "1.6.1_sbx_8",
   "custom": {
     "gear-builder": {
       "category": "converter",
-      "image": "dicom-metadata-classification:1.6.1"
+      "image": "dicom-metadata-classification:1.6.1_sbx_8"
```
Comment (NPann) [inline manifest.json:9]: Could revert back this change so that we only increase manifest version upon release?

File: OPHTHA_classifier.py [hunk 2]
```diff
@@ -109,31 +237,43 @@ def classify_OPHTHA(dcm_metadata, acquisition):
...
-        elif is_OCT_OP(acquisition.label):
-            oct_type = ['Fundus']
-            # print("Found match for Fundus OCT from is_OCT_OP match")
-        elif is_OCT_OPT(acquisition.label):
-            oct_type = ['Standard']
-            # print("Found match for Standard OCT from is_OCT_OPT match")
-
-        if oct_type:
-            classifications.update({"OCT Type": oct_type})
+        if octType == None:
```
Comment (NPann) [inline OPHTHA_classifier.py:246]: Nit: recommended using `is` in place of `==` when comparing to None.

File: OPHTHA_classifier.py [hunk 3]
```diff
@@ -5,100 +5,228 @@
...
+#         modality = ['FP']
+#         modalityType = ['Autofluorescence']
+#     elif is_COLOR(dicom_filepath):
+#         modality = ['FP']
+#         modalityType = ['Color']
+#     elif is_ICG:
+#         modality = ['FP']
+#         modalityType = ['Indocyanine Green']
 
+#     return modality
```
Comment (NPann) [inline OPHTHA_classifier.py:120]: Leaving these comments on purpose?

**Review by NPann — CHANGES_REQUESTED** [review]
Just requesting a change on the manifest version. The rest LGTM. 
A few other minor comments.

**Comment by jiavila** [comment]
https://github.com/flywheel-apps/GRP-3B/pull/36/files#diff-69d91eb184581109fbaaa14fc06d81d1R23-R27

I think there needs to be a space between two tests, or else it treats it as one. Can you double-check? Thanks!

**Comment by aldum1** [comment]
> https://github.com/flywheel-apps/GRP-3B/pull/36/files#diff-69d91eb184581109fbaaa14fc06d81d1R23-R27
> 
> @aldum1 I think there needs to be a space between two tests, or else it treats it as one. Can you double-check? Thanks!

I added spaces, but it looked like it was working without them as well

File: OPHTHA_classifier.py (hunk 2 above)
Comment (aldum1) [reply OPHTHA_classifier.py]: updated code to reflect suggestion. Thanks for pointing it out

**Comment by aldum1** [comment]
above issues are addressed

**Comment by jiavila** [comment]
> > https://github.com/flywheel-apps/GRP-3B/pull/36/files#diff-69d91eb184581109fbaaa14fc06d81d1R23-R27
> > @aldum1 I think there needs to be a space between two tests, or else it treats it as one. Can you double-check? Thanks!
> 
> I added spaces, but it looked like it was working without them as well

Good to know. Thanks, Alex!

**Review by NPann — APPROVED** [review]
Thanks Alex for addressing the comments. LGTM.

**Review by NPann — CHANGES_REQUESTED** [review]
@aldum1 , actually `python -m pytest` is failing on test_pt_classifier.py. Can you fix it so that we can merge it?

**Comment by kalebdfischer** [comment]
> @aldum1 , actually `python -m pytest` is failing on test_pt_classifier.py. Can you fix it so that we can merge it?

I fixed it in #35 (just merged to master)

**Review by kalebdfischer — APPROVED** [review]
LGTM

**Review by NPann — APPROVED** [review]

**Pull request merged**